client/src/features/app/app.component.js

```diff
@@ -20,11 +26,20 @@ export class App extends Component {
           console.log(error);
         });
       this.props.getLocation();
+      setup();
+      addResultsListener(this.resultsReceived);
+    }
+
+    resultsReceived(results) {
```

Comment:
can we use arrow functions in the class so that we dont have to bind the function in the constructor?

Reply:
+1